File: app/src/main/java/com/aar/app/wordsearch/features/gamethemeselector/GameThemeItem.java
package com.aar.app.wordsearch.features.gamethemeselector;

import android.arch.persistence.room.ColumnInfo;

public class GameThemeItem {
    @ColumnInfo(name = "id")
    private int mId;
    @ColumnInfo(name = "name")
    private String mName;
    @ColumnInfo(name = "words_count")
    private int mWordsCount;

    public int getId() {
        return mId;
    }

    public void setId(int id) {
        mId = id;
    }

    public String getName() {
        return mName;
    }

    public void setName(String name) {
        mName = name;
    }

    public int getWordsCount() {
        return mWordsCount;
    }

    public void setWordsCount(int wordsCount) {
        mWordsCount = wordsCount;
    }
}

File: app/src/main/java/com/aar/app/wsp/features/gamethemeselector/GameThemeItem.kt
package com.aar.app.wsp.features.gamethemeselector

import androidx.room.ColumnInfo

class GameThemeItem {
    @ColumnInfo(name = "id")
    var id = 0

    @ColumnInfo(name = "name")
    var name: String? = null

    @ColumnInfo(name = "words_count")
    var wordsCount = 0

}